runtime/doc/various.txt

```diff
@@ -208,15 +208,20 @@ g8			Print the hex values of the bytes used in the
 
 						  *:terminal* *:te*
 :te[rminal][!] {cmd}	Execute {cmd} with 'shell' in a |terminal-emulator|
-                        buffer.  Equivalent to: >
-			      :enew
-			      :call termopen('{cmd}')
-			      :startinsert
+			buffer.  Equivalent to: >
+				:enew
+				:call termopen('{cmd}')
+				:startinsert
 <
-                        See |jobstart()|.
-
-			To enter terminal mode automatically: >
-			      autocmd BufEnter term://* startinsert
+			See |jobstart()|.
+
+			To enter terminal mode automatically:
+>
+				augroup terminal_insert
```

Comment:
this is too much to expect every single place an autocmd is mentioned in the docs. So I left it out.